scripts/install

```diff
@@ -160,8 +160,8 @@ ynh_store_file_checksum --file="$final_path/config/jupyter_notebook_config.py"
 #=================================================
 
 # Set permissions to app files
-chown -R root: $final_path
-chown -R $admin: $final_path/.venv/share
```

Comment:
In fact, if we only change `/share' with `/lib' it works too, I'd prefer to restrict permissions as much as possible.


On the other hand, if a normal user (not the admin) tries to install a new package with pip, he will have the same problem as before, are you aware of that? It's not necessarily a problem to let only the admin install a new package, what do you think?

Reply:
You are right, It works. I guess it makes sens to restrict the install of packages to admin for now.